app/views/stats/index.html.erb

```diff
@@ -19,7 +19,7 @@
 
 <% @most_downloaded.each do |gem| %>
   <div class="stats__graph__gem">
-    <h3 class="stats__graph__gem__name"><%= gem.name %></h3>
+    <h3 class="stats__graph__gem__name"><a class="gems__gem" href="<%= rubygem_path(gem) %>"><%= gem.name %></a></h3>
```

Comment:
I thought too but it's not already used in `_rubygem.html.erb` where I took the link syntax.
Agree to use `link_to` in both?